src/main/java/com/snowplowanalytics/snowplow/tracker/Subject.java

```diff
@@ -53,15 +53,20 @@ public Subject(SubjectConfiguration subjectConfig) {
      * @param subject The subject from which the map is copied.
      */
     public Subject(Subject subject){
-      this.standardPairs.putAll(subject.getSubject());
+        standardPairs.putAll(subject.getSubject());
     }
 
+    /**
+     * @deprecated Use SubjectConfiguration class instead
```

Comment:
Should we also use the `@Deprecated` annotation on the methods in addition to the comment? I remember that in the Android tracker we also used the annotation. [Reading here](https://stackoverflow.com/a/9031850/3334072) it seems that the annotation is important for compilers to generate warnings.